site/zenodo_rdm/templates/semantic-ui/zenodo_rdm/support.html

```diff
@@ -0,0 +1,52 @@
+{%- extends "invenio_app_rdm/frontpage.html" %}
+
+{%- block page_header %}
+    {%- include "zenodo_rdm/header_frontpage.html" %}
+{%- endblock page_header %}
+
+{%- set user = current_user.is_authenticated and current_user %}
+{%- set userData = user and { 'email': user.email, 'full_name': user.user_profile.full_name }%}
+
+{% block first_section %}
+    <div id="contact-page" class="ui one column grid container">
+        <div class="row">
+            <div class="column">
+                <div class="panel-body mt-5">
+                    <h2 class="ui header mb-10">Contact us</h2>
+                    <p>Before making a request, you can have a look at our <a href="http://help.zenodo.org">FAQ</a> and <a href="http://about.zenodo.org">other resources</a> for more detailed information about the operation, features and goals of Zenodo.</p>
```

Comment:
links are still pointing to `zenodo.org`. Could be passed by config.

However many links in `zenodo-rdm` are currently pointing to `zenodo.org`

Reply:
we agreed to keep it for now